src/engine/strat_engine/filesystem.rs

```diff
@@ -94,20 +96,22 @@ impl StratFilesystem {
 
     /// Get the mount_point for this filesystem
     /// TODO Replace this code with something less brittle
-    pub fn get_mount_point(&self) -> EngineResult<PathBuf> {
+    pub fn get_mount_point(&self) -> EngineResult<Option<PathBuf>> {
         let output = Command::new("df")
             .arg("--output=target")
             .arg(&self.devnode())
             .output()?;
         if output.status.success() {
             let output_str = String::from_utf8_lossy(&output.stdout);
             if let Some(mount_point) = output_str.lines().last() {
-                return Ok(PathBuf::from(mount_point));
+                return Ok(Some(PathBuf::from(mount_point)));
+            } else {
+                return Ok(None);
             }
         }
-        let err_msg = format!("Failed to get filesystem mountpoint at {:?}",
-                              self.devnode());
-        Err(EngineError::Engine(ErrorEnum::Error, err_msg))
+        Err(EngineError::Engine(ErrorEnum::Error,
+                                format!("Failed to get filesystem mountpoint at {:?}",
```

Comment:
"at" might be a very non-descriptive preposition. I think there is probably a clearer error string.

Reply:
Ok.  How about "for"?

format!("Failed to get filesystem mountpoint for {:?}", self.devnode())))

If this doesn't work, could you please suggest what you'd like to see.  The error message seems clear to me.